src/test/ui/panic-brace.stderr

```diff
@@ -0,0 +1,71 @@
+warning: Panic message contains a brace
+  --> $DIR/panic-brace.rs:5:5
+   |
+LL |     panic!("here's a brace: {");
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+   = note: `#[warn(panic_fmt)]` on by default
+   = note: This message is not used as a format string, but will be in a future Rust version
+help: add a "{}" format string to use the message literally
+   |
+LL |     panic!("{}", "here's a brace: {");
+   |            ^^^^^
+
+warning: Panic message contains a brace
+  --> $DIR/panic-brace.rs:6:5
+   |
+LL |     std::panic!("another one: }");
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+   = note: This message is not used as a format string, but will be in a future Rust version
+help: add a "{}" format string to use the message literally
+   |
+LL |     std::panic!("{}", "another one: }");
+   |                 ^^^^^
+
+warning: Panic message contains an unused formatting placeholder
+  --> $DIR/panic-brace.rs:7:18
+   |
+LL |     core::panic!("Hello {}");
+   |                  ^^^^^^^^^^
+   |
+   = note: This message is not used as a format string when given without arguments, but will be in a future Rust version
+help: add the missing argument(s)
+   |
+LL |     core::panic!("Hello {}", argument);
+   |                            ^^^^^^^^^^
+help: or add a "{}" format string to use the message literally
+   |
+LL |     core::panic!("{}", "Hello {}");
+   |                  ^^^^^
+
+warning: Panic message contains an unused formatting placeholder
+  --> $DIR/panic-brace.rs:8:20
+   |
+LL |     assert!(false, "{:03x} bla");
+   |                    ^^^^^^^^^^^^
+   |
+   = note: This message is not used as a format string when given without arguments, but will be in a future Rust version
+help: add the missing argument(s)
+   |
+LL |     assert!(false, "{:03x} bla", argument);
+   |                                ^^^^^^^^^^
+help: or add a "{}" format string to use the message literally
+   |
+LL |     assert!(false, "{}", "{:03x} bla");
+   |                    ^^^^^
```

Comment:
I feel that the second suggestion should be for `assert!(false, "{{:03x}} bla");` instead.

Reply:
That would literally output `"{{:03x}} bla"`, with double braces. See my other comment.